### PR TITLE
Build default HTML for snippets

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Some part of me thinks this work belongs in Composer, because that is the only place it is every going to be used. Meanwhile, all CAPI requests will carry this additional payload even if they don't need it.

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
Once again this builds correctly on my machine but times out in CI 😩

File: app/util/AtomElementBuilders.scala [hunk 1]
```diff
@@ -63,14 +64,49 @@ object AtomElementBuilders {
   }
 
   def buildHtml(atomType: AtomType, atomData: AtomData): Option[String] = atomType match {
-    case AtomType.Storyquestions => buildStoryQuestionsHtml(atomData)
+    case AtomType.Storyquestions => buildStoryQuestionsHtml(atomData.asInstanceOf[AtomData.Storyquestions].storyquestions)
```
Comment (tomrf1) [inline app/util/AtomElementBuilders.scala]: Not really an issue, but you can avoid using `asInstanceOf` by matching on the atomData, e.g. https://github.com/guardian/story-manager/blob/master/app/models/Atom.scala#L12

Comment (regiskuckaertz) [inline app/util/AtomElementBuilders.scala]: Brilliant! I did that.

**Comment by regiskuckaertz** [comment]
Ping. Is everybody happy?

**Comment by tomrf1** [comment]
👍 I'm happy with the code, and using this as a quick fix.
But I agree with your comment that this belongs in composer, not capi.

**Comment by regiskuckaertz** [comment]
Thanks! Yep, I'll card this in Trello and will move that to composer in short time.

**Pull request merged**